src/framework/theme/components/menu/menu.service.ts

```diff
@@ -4,14 +4,17 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  */
 
-import { Injectable } from '@angular/core';
-import { Location } from '@angular/common';
-import { Params } from '@angular/router';
-import { Observable, BehaviorSubject, ReplaySubject, Subject } from 'rxjs';
-import { share } from 'rxjs/operators';
-import { isUrlPathContain, isUrlPathEqual } from './url-matching-helpers';
-
-export interface NbMenuBag { tag: string; item: NbMenuItem }
+import {Injectable} from '@angular/core';
```

Comment:
can we revert all unnecessary formatting changes?

Reply:
@nnixaa sure please. I noticed it after pull request.

Reply:
@nnixaa reverted unnecessary formatting changes.